apps/api/v2/src/ee/bookings/2024-08-13/controllers/e2e/user-bookings.e2e-spec.ts

```diff
@@ -1936,292 +1939,468 @@ describe("Bookings Endpoints 2024-08-13", () => {
         eventTypeWithAllLocationsId = createdEventType.id;
       });
 
-      it("can book with cal video location", async () => {
-        const bookingBody: CreateBookingInput_2024_08_13 = {
-          start: new Date(Date.UTC(2040, 0, 9, 13, 0, 0)).toISOString(),
-          eventTypeId: eventTypeWithAllLocationsId,
-          attendee: {
-            name: "Mr Proper",
-            email: "mr_proper@gmail.com",
-            timeZone: "Europe/Rome",
-            language: "it",
-          },
-          location: {
-            type: "integration",
-            integration: "cal-video",
-          },
-        };
+      describe("can book with multiple locations", async () => {
```

Comment:
I would rename it "can book with different locations" because you can book with only 1 location